### PR TITLE
Feat/scout 260 dashboard

### PR DESCRIPTION
**Pull request by dejwi**
### Task Description
[SCOUT-260](https://playmakerpro.atlassian.net/browse/SCOUT-81?atlOrigin=eyJpIjoiMTU4OWJlMDQxZjk2NDI4ZGJjODk4Y2M4NTZhZDlhYjUiLCJwIjoiaiJ9)

new dashboard
- in scout organization players count is 0 because of our team endDate null issue

### Requires related backend pr

### CODE REVIEW

**Comment by dejwi** [comment]
forgot to mention that I'll add translations at the end in case of some changes

File: src/modules/dashboard/ListDataCard.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  Grid,
+} from '@mui/material'
+
+import { OptionalLink } from '@/components/links/optional-link'
+
+import { StyledAvatar } from './StyledAvatar'
+
+interface IListDataCardProps {
+  title: string
+  items: JSX.Element | JSX.Element[] | string
```
Comment (jakubfiglak) [inline src/modules/dashboard/ListDataCard.tsx:15]:
```suggestion
  items: React.ReactNode
```

This is basically the same type you've written here by hand

File: src/modules/dashboard/layouts/playmaker-scout.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <ListDataCard
+        icon={<OrganizationsIcon />}
+        title={t('ORGANIZATIONS')}
+        subheader="Organizacje z aktywnymi subskrypcjami"
+        items={
+          organizations?.map(({ name }) => (
+            <Grid item xs={12}>
+              <Typography fontWeight="Bold">{name}</Typography>
+            </Grid>
+          )) || 'ERROR: DATA NOT RECIVED'
```
Comment (jakubfiglak) [inline src/modules/dashboard/layouts/playmaker-scout.tsx:34]: Do we want to display an error like this?

File: src/modules/dashboard/layouts/scout-organization.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                value={`${rating} - ${
+                  match ? getBasicMatchName(match) : ''
+                } (${getDocumentNumber({
+                  docNumber,
+                  createdAt,
+                })})`}
+                href={`/notes/${id}`}
+              />
+            ),
+          ) || 'ERROR: NO DATA'
```
Comment (jakubfiglak) [inline src/modules/dashboard/layouts/scout-organization.tsx:72]: Do we want to display an error like this?

File: src/modules/dashboard/layouts/scout-organization.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+                value={`${finalRating} - ${
+                  match ? getBasicMatchName(match) : ''
+                } (${getDocumentNumber({
+                  docNumber,
+                  createdAt,
+                })})`}
+                href={`/reports/${id}`}
+              />
+            ),
+          ) || 'ERROR: NO DATA'
```
Comment (jakubfiglak) [inline src/modules/dashboard/layouts/scout-organization.tsx:94]: And here?

Comment (dejwi) [reply src/modules/dashboard/layouts/scout-organization.tsx]: At first I thought that if we don't have anything here it means that there's a underlaying issue but now thinking about it if user has slower internet connections he'll see all this errors so I removed them

File: src/components/links/optional-link.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+
+export const OptionalLink = ({
+  href,
+  children,
+  passhref,
```
[the remark this answers is not in the document]
Comment (dejwi) [reply src/components/links/optional-link.tsx]: if you mean `passhref` I think it just remained from some of my playing around and I forgot to remove it

**Review by jakubfiglak — COMMENTED** [review]
Code LGTM, but we are waiting for the final UI, aren't we?

**Comment by dejwi** [comment]
> Code LGTM, but we are waiting for the final UI, aren't we?

yes

**Review by jakubfiglak — APPROVED** [review]

**Pull request merged**